_redirects.ejs

```diff
@@ -0,0 +1,2 @@
+/index.html / 301
+/* /index.html 301
```

Comment:
If this line worked, wouldn't all our image, script, & CSS requests fail?

Reply:
If netlify's server finds a static file it will return it anyway. If we wanted to override all files we would need to pass an exclamation mark e.g. `300!`. You can test it on deploy preview

Reply:
TIL, thanks!